### PR TITLE
Read-only (for now) UI to browse applications and their settings

### PR DESCRIPTION
**Pull request by jrgp**
Subsequent PRs will have the ability to edit the settings for each app, similar to how plans/templates are currently edited.

### CODE REVIEW

File: src/iris_frontend/app.py [hunk 1]
```diff
@@ -327,6 +327,18 @@ def validate():
         })
 
 
+@app.route('/applications/')
+@app.route('/applications/<name>')
+@login_required
+def applications(name=None):
+    user = current_user.id
+    applications = json.loads(client.get('applications').data)
+    if name:
```
Comment (shoxv) [inline src/iris_frontend/app.py]: I'm a python scrub but is this if condition needed?

Comment (jrgp) [inline src/iris_frontend/app.py]: Yes, as it determines which template to render (application.html vs applications.html)

File: src/iris_frontend/static/js/iris.js [hunk 2]
```diff
@@ -1752,6 +1752,65 @@ iris = {
...
+        return;
+      }
+      iris.changeTitle('Application ' + app.name);
+      $.getJSON(this.data.url + application + '/quota').done(function(response) {
+        app.quota = response;
+        self.data.$page.html(template(app));
+      }).fail(function() {
+        // When quota does not exist, that endpoint 404s even though application is real
+        app.quota = null;
+        self.data.$page.html(template(app));
```
Comment (shoxv) [inline src/iris_frontend/static/js/iris.js]: Do we wanna show a 404 page here, or some kind of error to the user? I'm not sure what the page looks like so what renders here if the api 404s?

Comment (jrgp) [inline src/iris_frontend/static/js/iris.js]: This is just getting settings for the quotas. If it 404s the box for quota (in application.html) just says "no quota", with the rest of the application page being displayed.

File: src/iris_frontend/templates/application.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    <h3>{{name}}</h3>
+    {{#if owners}}
+    <span class="light">Owners: {{#each owners}}<span class="label label-info">{{this}}</span> {{/each}}</span><br>
+    {{/if}}
+    <span class="light">Variables: {{#each variables}}<span class="label label-info">{{this}}</span> {{/each}}</span>
+  </div>
+  <div class="module">
+    <h4>Context Template</h4>
+    <div class="application-setting">
+      <pre>{{context_template}}</textarea>
```
Comment (shoxv) [inline src/iris_frontend/templates/application.html]: incorrect closing tag here

Comment (jrgp) [inline src/iris_frontend/templates/application.html]: Oops

Comment (jrgp) [inline src/iris_frontend/templates/application.html]: Just updated PR to fix this.

File: src/iris_frontend/app.py (hunk 1 above)
Comment (shoxv) [inline src/iris_frontend/app.py]: whoops, i didnt see the s. lol

**Pull request merged**